Dockerfile

```diff
@@ -26,15 +26,18 @@ RUN apt-get update && \
 
 # sudo mount -o bind ... bundle/*
 ADD bundle/spark /spark/
-ADD bundle/engine/python/sourced/engine/ /packages/sourced/engine/
-ADD bundle/ml/sourced/ml/ /packages/sourced/ml/
+ADD bundle/engine/python /packages/sourced/engine/
+ADD bundle/ml /packages/sourced/ml/
```

Comment:
Depends on what you bind mount, doesn't it. The default code assumes the roots of the projects and it should stay intact.